aexpect/remote.py

```diff
@@ -383,8 +383,7 @@ def remote_login(client, host, port, username, password, prompt, linesep="\n",
     :return: A RemoteSession object.
     """
     extra_params = []
-    if verbose:
-        extra_params.append("-vv")
+    extra_params.append(f"{'-vv' if verbose else ''}")
```

Comment:
IIUC, this change is adding an empty string to the list in case of normal run (verbose=false). While this is valid, I find odd having a empty value inside a "list of extra parameters". Unless I'm missing something, the fix itself is on the line 407. And it will work even with this current logic here. So, is there any reason for you adding this empty string here?

Reply:
Also, I know that is not related to your change, but if you dont mind, could please, take this opportunity to add the 'verbose' argument to this method docstring, since it is missing?

Reply:
Yes, The change in L386 is not necessary.